### PR TITLE
Book display

### PR DESCRIPTION
**Pull request by XLuna24**
Displays the list of books or movies according to users search. It Display's both Books and Movies in small boxes with a title and description.

### CODE REVIEW

File: src/main/webapp/home.js [hunk 1]
```diff
@@ -1,17 +1,75 @@
...
             contentList.appendChild(createContentElement(info));
         })
     });
 }
 
-//Creates an li to display the title of the movies
+function loadBooks() {
+    const search = document.getElementById("search").value;
+
+    fetch('/books/search?query=' + search + "&pageNumber=1")
```
Comment (henryksloan) [inline src/main/webapp/home.js:42]: Book pages actually start at zero. For now, change this to zero so it shows the right results. It definitely makes more sense to query pages as they're shown to the user (i.e. 1 is first), so I'll change my code once we get a chance to talk about it.

File: src/main/webapp/home.js [hunk 2]
```diff
@@ -1,17 +1,75 @@
...
+    const description = document.createElement('div');
+    description.className = 'description-box';
+    description.innerText = info.description ? info.description : info.overview;
+
+    contentElement.appendChild(title);
+    contentElement.appendChild(description);
     return contentElement;
+}
+
+//Display's wether the movies or books list appears
```
Comment (henryksloan) [inline src/main/webapp/home.js:69]: Displays whether

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -12,18 +12,24 @@
           <h1>Mediaphile</h1>
           <div class="search-container">
                   <input type="text" placeholder="Search.." id="search">
-                  <button class="button" onclick="loadMovies()">Submit</button>
+                  <button class="button" onclick="loadContent()">Submit</button>
+                  <label>in</label>
+                  <select id="selection" class="button">
+                      <option value="movie">Movies</option>
+                      <option value="book">Books</option>
+                      <option value="all">All</option>
```
Comment (henryksloan) [inline src/main/webapp/index.html:20]: I imagine "All" would be on top/default, but don't change this for now. We'll all discuss that as we think about the search journey.

Comment (TheGuyWhoCodes) [reply src/main/webapp/index.html:20]: using all will bring up some challenges for standardizing how we interpret search results ie a book entity and a movie entity from a search result are different from each other.

Comment (XLuna24) [reply src/main/webapp/index.html]: has been switched to movies/books and made default so later on we can add in the user search

File: src/main/webapp/home.js [hunk 4]
```diff
@@ -1,17 +1,83 @@
...
     return contentElement;
+}
+
+//Display's whether the movies or books list appears
+function displayContent(content) {
+    document.getElementById("movie-list").style.display = "none";
+    document.getElementById("book-list").style.display = "none";
+
+    document.getElementById(content).style.display = "block";
 }
```
Comment (kirantumkur) [inline src/main/webapp/home.js:83]: This code has a lot of "const" variables which are being reassigned values. Please verify that "const" is being used correctly here.

**Pull request merged**